src/tests/dictionaryConcepts/container/DictionaryConcepts.test.jsx

```diff
@@ -618,5 +620,84 @@ describe('Test suite for dictionary concepts components', () => {
         expect(spy).toHaveBeenCalled();
       });
     });
+
+    describe('handleRetireConcept', () => {
+      let instance;
+      let concept;
+      let retire;
+      beforeEach(() => {
+        instance = wrapper.find('DictionaryConcepts').instance();
+        retireMockProps.retireCurrentConcept.mockClear();
+        retireMockProps.addReferenceToCollection.mockClear();
+        retireMockProps.deleteReferenceFromCollection.mockClear();
+        [concept] = props.concepts;
+        retire = true;
+      });
+
+      it('calls retireCurrentConcept, deleteReferenceFromCollection and addReferenceToCollection with the right arguments', async () => {
+        await instance.handleRetireConcept(concept.id, retire);
+        expect(retireMockProps.retireCurrentConcept).toHaveBeenCalledWith(concept.url, retire);
+        expect(retireMockProps.deleteReferenceFromCollection).toHaveBeenCalledWith(
+          props.match.params.type,
+          props.match.params.typeName,
+          props.match.params.collectionName,
+          [concept.version_url],
+        );
+        expect(retireMockProps.addReferenceToCollection).toHaveBeenCalledWith(
+          props.match.params.type,
+          props.match.params.typeName,
+          props.match.params.collectionName,
+          [concept.url],
+        );
+      });
+
+      it('does not call deleteReferenceFromCollection and addReferenceToCollection when retire fails', async () => {
```

Comment:
Your descriptions should follow the same convention as the rest of the code base.

Reply:
Working on this.

Reply:
This has now been addressed.